.copr/Makefile

```diff
@@ -0,0 +1,17 @@
+.copr/yggdrasil.spec srpm: COMMIT = $(shell git rev-parse HEAD)
+
+srpm: SOURCEDIR := $(shell rpmbuild -E %_sourcedir)
+srpm: SRCRPMDIR := $(shell rpmbuild -E %_srcrpmdir)
+
+.PHONY: deps
+deps:
+	-dnf install -y git
+
+.copr/yggdrasil.spec: .copr/yggdrasil.spec.in
+	sed -e 's/@COMMIT@/$(COMMIT)/g' $^ > $@.tmp && mv $@.tmp $@
+
+.PHONY: srpm
+srpm: deps .copr/yggdrasil.spec
+	git archive --prefix yggdrasil-$(COMMIT)/ --format tgz $(COMMIT) --output $(SOURCEDIR)/yggdrasil-$(COMMIT).tar.gz
+	rpmbuild -bs --verbose $(spec)
+	mv $(SRCRPMDIR)/*.src.rpm $(outdir)
```

Comment:
`$(spec)` and `$(outdir)` are declared as make variables by COPR when it invokes this Makefile. The [COPR documentation on the make srpm source method](https://docs.pagure.org/copr.copr/user_documentation.html#make-srpm) says that the Makefile's target is invoked like this:

```
make -f <cloned_repodir>/.copr/Makefile srpm outdir="<outdir>" spec="<spec_path>"

```

> All *.src.rpm file from $(SRCRPMDIR) are moved (not only generated file).

There will only be one SRPM in `$(SRCRPMDIR)`. Each time this is invoked by COPR, it is run in a new directory. There's not really much difference between moving and copying, since the working directory is destroyed afterwards.

Reply:
OK